test/plugins/graphql.spec.js

```diff
@@ -521,21 +604,25 @@ describe('Plugin', () => {
 
         it('should handle calling low level APIs directly', done => {
           const source = `query MyQuery { hello(name: "world") }`
-          const document = graphql.parse(source)
 
           agent
             .use(traces => {
               const spans = sort(traces[0])
 
-              expect(spans).to.have.length(3)
-              expect(spans[0]).to.have.property('service', 'test-graphql')
+              expect(spans).to.have.length(6)
               expect(spans[0]).to.have.property('name', 'graphql.query')
-              expect(spans[0]).to.have.property('resource', 'query MyQuery')
-              expect(spans[0].meta).to.have.property('graphql.document', source)
```

Comment:
Why were all instances of this test completely removed?

Reply:
I’m not sure I entirely understand what you mean by “all instances of this test”, but in this case I probably unconsciously removed it for being noise that was already tested elsewhere. Do you want me to revert this?

Reply:
You are correct, it is redundant. I missed it because I was looking at just the diff.